src/app/frontend/logs/logstoolbar/logstoolbar_controller.js

```diff
@@ -130,7 +130,14 @@ export default class LogsToolbarController {
    * @return {!backendApi.ReplicationControllerPodWithContainers|undefined}
    * @private
    */
-  findPodByName_(array, name) { return array.find((element) => element.name === name); }
+  findPodByName_(array, name) {
```

Comment:
That's a very nice article. I've read it and it has very good point. Overriding is dangerous. It's safer to rely on some 3rd party libraries. Polyfill with closure compiler is also good idea. We should think of that.